internal/datacoord/server.go

```diff
@@ -295,12 +296,23 @@ func (s *Server) Start() error {
 	s.startServerLoop()
 	Params.DataCoordCfg.CreatedTime = time.Now()
 	Params.DataCoordCfg.UpdatedTime = time.Now()
-	atomic.StoreInt64(&s.isServing, ServerStateHealthy)
-	logutil.Logger(s.ctx).Debug("startup success")
 
 	return nil
 }
 
+// UpdateStateCode updates dataCoord's state code
+// TODO:: use internalpb.StateCode instead of ServerState
+func (s *Server) UpdateStateCode(code internalpb.StateCode) {
+	switch code {
+	case internalpb.StateCode_Initializing:
```

Comment:
const (
	StateCode_Initializing StateCode = 0
	StateCode_Healthy      StateCode = 1
	StateCode_Abnormal     StateCode = 2
)
I think above variables are redundant, why not use following variables directly?

const (
	// ServerStateStopped state stands for just created or stopped `Server` instance
	ServerStateStopped ServerState = 0
	// ServerStateInitializing state stands initializing `Server` instance
	ServerStateInitializing ServerState = 1
	// ServerStateHealthy state stands for healthy `Server` instance
	ServerStateHealthy ServerState = 2
)